core/server/services/labs.js

```diff
@@ -6,6 +6,14 @@ const i18n = require('../../shared/i18n');
 const logging = require('../../shared/logging');
 const settingsCache = require('../services/settings/cache');
 
+// NOTE: this allowlist is meant to be used to filter out any unexpected
+//       input for the "labs" setting value
+const WRITABLE_KEYS_ALLOWLIST = [
```

Comment:
@ErisDS I've put the allowlist in the service layer with a mindset that it's the best place to keep things central when it comes to reuse across API serialization, model, and importer layers. Was wondering if you have any thoughts on better placement of data like this?

Reply:
This seems fine to me for now. Maybe a better location will come to mind when I start to use it in anger - but this seems sensible.